src/contracts/fund/hub/Hub.ts

```diff
@@ -4,6 +4,16 @@ import { Contract } from '../../../Contract';
 import { Environment } from '../../../Environment';
 import { Address } from '../../../Address';
 import { toDate } from '../../../utils/toDate';
+import { ValidationError } from '../../../errors/ValidationError';
+import { sameAddress } from '../../../utils/sameAddress';
+import { validate } from '../../../utils/validate';
+
+export class OnlyCreator extends ValidationError {
+  constructor(message?: string) {
+    const msg = message ? message : 'Only creator can do this';
+    super(msg);
```

Comment:
Does this go through the linter? Usually it expects super() as the first call in the constructor.

Reply:
It could also be one-lined like so: `super(message || 'Only the creator can do this')`.